packages/protocol/contracts/rrp/authorizers/Api3RequesterRrpAuthorizer.sol

```diff
@@ -29,6 +29,22 @@ contract Api3RequesterRrpAuthorizer is
         override(MetaAdminnable, RankedAdminnable, IRankedAdminnable)
         returns (uint256)
     {
-        return MetaAdminnable.getRank(adminnedId, admin);
+        return MetaAdminnable.getRank(bytes32(0), admin);
+    }
+
+    /// @notice Called by an admin of higher rank to set the rank of an admin of
+    /// lower rank
+    /// @dev Use this instead of setRank(bytes32 adminnedId,address targetAdmin, uint256 newRank)
+    /// @param targetAdmin Target admin address
+    /// @param newRank Rank to be set
+    function setRank(address targetAdmin, uint256 newRank) public {
```

Comment:
This needs to be added too `IApi3RequesterRrpAuthorizer`

Reply:
My bad, tend to forget to update the interfaces and have to remind myself.I've updated it